### PR TITLE
Feature/turbo upi

### PR DESCRIPTION
**Pull request by shibu000**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java [hunk 1]
```diff
@@ -176,4 +188,67 @@ public void onExternalWalletSelected(String walletName, PaymentData paymentData)
...
+                    reply.put("data", "");
+                }else {
+                    reply.put("data", toJsonString(upiAccounts));
+                }
+                //sendReply(reply);
+            }
+
+            @Override
+            public void onError(@NonNull Error error) {
+              //  pendingResult.error(error.getErrorCode(), error.getErrorDescription(), toJsonString(error));
```
Comment (vivekshindhe) [inline android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java]: @shibu000 we don't need a reply here?

File: android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java [hunk 2]
```diff
@@ -176,4 +188,67 @@ public void onExternalWalletSelected(String walletName, PaymentData paymentData)
...
+        } catch (ClassNotFoundException e) {
+            // Class not found, so it doesn't exist
+            reply.put("isTurboPluginAvailable", false);
+            sendReply(reply);
+            return false;
+        }
+    }
+
+    private String toJsonString(Object object){
+        return this.gson.toJson(object);
```
Comment (vivekshindhe) [inline android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java:272]: When an upiAccount object is converted to json, does it also include the vpa object? @shibu000

File: android/src/main/java/com/razorpay/razorpay_flutter/RazorpayFlutterPlugin.java [hunk 3]
```diff
@@ -23,6 +23,9 @@ public class RazorpayFlutterPlugin implements FlutterPlugin, MethodCallHandler,
     private RazorpayDelegate razorpayDelegate;
     private ActivityPluginBinding pluginBinding;
     private static String CHANNEL_NAME = "razorpay_flutter";
+    Map<String, Object> _arguments;
```
Comment (vivekshindhe) [inline android/src/main/java/com/razorpay/razorpay_flutter/RazorpayFlutterPlugin.java:27]: @shibu000 any reason for this variable? Can we look into inlining the requirement in the switch condition itself?

File: lib/checkout.dart [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/services.dart';
+import 'package:razorpay_flutter/upi_turbo.dart';
+
+class CheckOut{
```
Comment (vivekshindhe) [inline lib/checkout.dart:4]: @shibu000 Why was this class created? We already have razorpay_flutter.dart. Can we not reuse the same class?

File: android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java (hunk 1 above)
Comment (shibu000) [reply android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java]: @vivekshindhe Have added it, Done.

File: android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java (hunk 2 above)
Comment (shibu000) [reply android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java:272]: Yes @vivekshindhe

File: android/src/main/java/com/razorpay/razorpay_flutter/RazorpayFlutterPlugin.java (hunk 3 above)
Comment (shibu000) [reply android/src/main/java/com/razorpay/razorpay_flutter/RazorpayFlutterPlugin.java:27]: @vivekshindhe This variable is used for holding parameter value which we are getting from dart to the native side, As we are using this variable multiple places inside the function, we thought of avoiding inlining initialization.

File: lib/checkout.dart (hunk 4 above)
Comment (shibu000) [reply lib/checkout.dart:4]: @vivekshindhe The main reason is to follow the same function signature that we have approved in the API council review to expose it to the merchant.

Comment (vivekshindhe) [reply lib/checkout.dart:4]: This is not it. The naming convention is wrong. And we will continue to use whatever class already exists. the name checkout is of the class in that SDK. It's not mandatory to use it. razorpay.upiTurbo.{funcName} is also reviewed approved. Please change this. We cannot ask the merchant to initialize another object. This beats the purpose of using it as a part of the base Checkout SDKs.

**Comment by Imgkl** [comment]
This feature was reverted? Couldn't find the `initUpiTurbo` method.

**Pull request merged**